eng/pipelines/templates/jobs/archetype-sdk-tests.yml

```diff
@@ -10,26 +10,26 @@ jobs:
           OSVmImage: 'ubuntu-18.04'
           vcpkg.deps: 'curl[ssl]'
           VCPKG_DEFAULT_TRIPLET: 'x64-linux'
-          build.args: ' -DTRANSPORT_CURL=ON'
+          build.args: ' -DTRANSPORT_CURL=ON -DAZ_PLATFORM_IMPL=POSIX'
```

Comment:
With this change, if we don't fix https://github.com/Azure/azure-sdk-for-c/issues/1290, will live tests now start to fail?

Reply:
yes, live test will fail on next run unless we merge this fix.
#1290 would just help to know which dependency is missing, but platform will still be missing for live tests